keras/layers/regularization/activity_regularization.py

```diff
@@ -29,6 +29,8 @@ class ActivityRegularization(Layer):
     Args:
       l1: L1 regularization factor (positive float).
       l2: L2 regularization factor (positive float).
+      Note: Both l1 and l2 also accepts integers and
```

Comment:
Python is not a typed language. We should remove this note. Users are expected to pass floats.